EmailValidation/EmailAttribute.cs

```diff
@@ -28,21 +28,28 @@
 
 namespace EmailValidation
 {
-	[AttributeUsage (AttributeTargets.Property | AttributeTargets.Field, AllowMultiple = false)]
-	public sealed class EmailAttribute : ValidationAttribute
-	{
-		public EmailAttribute (bool allowTopLevelDomains = false, bool allowInternational = false)
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field, AllowMultiple = false)]
+    public sealed class EmailAttribute : ValidationAttribute
+    {
+        public EmailAttribute() : this(false, false) { }
+
+        public EmailAttribute(bool allowTopLevelDomains) : this(allowTopLevelDomains, false) { }
+
+        public EmailAttribute (bool allowTopLevelDomains, bool allowInternational)
 		{
 			AllowTopLevelDomains = allowTopLevelDomains;
 			AllowInternational = allowInternational;
 		}
 
-		public bool AllowTopLevelDomains { get; set; }
+		public bool AllowTopLevelDomains { get; }
 
-		public bool AllowInternational { get; set; }
+		public bool AllowInternational { get; }
```

Comment:
Ez jó döntés volt, hiszen a konstruktorban már alapból beállítjuk ezeket a változókat, nincs szükség setre (inkonzisztenciát okozhat)